Integrations/integration-fireeye.yml

```diff
@@ -235,7 +264,7 @@ script:
             var filename = getFileName(args);
             result = {Type: 9, FileID: saveFile(response.Bytes), File: filename, Contents: filename};
             break;
-        case 'fe-submit':
+        case 'fe-submit-file':
```

Comment:
@sgosai I think it might be better to keep this as before for backwards compatibility

Reply:
@hirshbergshachar, integration has been updated.